sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,248 @@
+## APIs:
+class ServiceBusClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
```

Comment:
I think this has carried over from eventhubs - from what I can tell the Azure portal always lists it as 'host name'. Should we match the portal?

Reply:
this would need service side input

Reply:
It seems to be called host name in the portal for event hubs as well